launcher/res/qml/icons.js

```diff
@@ -0,0 +1,8 @@
+.pragma library
+
+var map = {
```

Comment:
Aren't these hardcoded in? How does the user provide an icon for, say, Blender?

Reply:
They are hardcoded but like I said I didn't found different way to be able use both awesome and png icons. I like easy and dynamic approches so if you know how achieve same or better result with AVALON_ICON_ROOT like you described I agree, but don't know how to because all my attempts with similar approach failed... If you know how to I agree with canceling this PR and creating new one :)